### PR TITLE
Migrating sequences from clarin5 to clarin7

### PR DESCRIPTION
**Pull request by PavolStefanec**
Migration results are tested immediately after the sequence value is set. Is it enough???

| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: const.py [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
-# fe_port = ":4000"
-fe_port = None
-# be_port = ":8080"
-be_port = None
+host = "localhost"
+# host = "dev-5.pc"
+fe_port = ":4000"
+# fe_port = None
+be_port = ":8080"
+# be_port = None
```
Comment (MajoBerger) [inline const.py:17]: you should not push changes in this file. rebase it

File: main.data_pump.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import logging
 import argparse
 import sys
```
Comment (MajoBerger) [inline main.data_pump.py:3]: why space

**Review by MajoBerger — CHANGES_REQUESTED** [review]
not acceptable

File: main.data_pump.py (hunk 2 above)
Comment (MajoBerger) [reply main.data_pump.py:3]: there is still space @PavolStefanec Milan by sa nepotešil teda

**Review by MajoBerger — CHANGES_REQUESTED** [review]
remove the space.. or Milan will tell you to

**Review by MajoBerger — APPROVED** [review]
finally

File: const.py (hunk 1 above)
Comment (milanmajchrak) [reply const.py]: @PavolStefanec I like that space in the comment. Nice job, that was kind of designer move. But we must listen @MajoBerger

Comment (MajoBerger) [reply const.py]: @milanmajchrak I will not forget this. And you also replied to wrong conversation. And it also was not a designer move, the space was between imports.

**Review by milanmajchrak — APPROVED** [review]

**Comment by milanmajchrak** [comment]
I tested it and it works.

**Pull request merged**